packages/markdown/parseMarkdown.js

```diff
@@ -419,10 +419,20 @@ function createRender(context) {
             }
             return undefined;
           },
+
           renderer(token) {
-            return `<aside class="MuiCallout-root MuiCallout-${token.severity}">${this.parser.parse(
-              token.tokens,
-            )}\n</aside>`;
+            return `<aside class="MuiCallout-root MuiCallout-${token.severity}">
+            ${
+              ['info', 'success', 'warning', 'error'].includes(token.severity)
+                ? [
+                    '<svg focusable="false" aria-hidden="true" viewBox="0 0 24 24" data-testid="ContentCopyRoundedIcon">',
+                    `<use class="MuiCode-copied-icon" xlink:href="#${token.severity}-icon" />`,
```

Comment:
Now on any page, these SVG get loaded. This can't be tree-shaked. e.g. view-source:https://master--material-ui.netlify.app/blog/mui-x-mid-v6-features/.

I wonder what's best. Per https://cloudfour.com/thinks/svg-icon-stress-test/, we could use inline SVG, it doesn't see much different than the symbol sprite for performance, so maybe better.